lib/resources/styles.css

```diff
@@ -16,6 +16,13 @@ body {
   margin: 0;
   padding: 0;
   height: 100%;
+  width: 100%;
+  overflow: hidden;
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
```

Comment:
This seems somewhat drastic to change every element to box sizing. From very brief googling (https://css-tricks.com/inheriting-box-sizing-probably-slightly-better-best-practice/) this does seem like a thing that people do.

Reply:
Yes.  Doing this seemed to make correcting some alignment problems for mobile a lot easier.